jsonrpc/websocket.go

```diff
@@ -2,21 +2,24 @@ package jsonrpc
 
 import (
 	"context"
+	"errors"
 	"io"
 	"net/http"
 	"strings"
-	"sync/atomic"
 	"time"
 
 	"github.com/NethermindEth/juno/utils"
 	"github.com/coder/websocket"
+	"golang.org/x/sync/semaphore"
 )
 
 const (
 	closeReasonMaxBytes = 125
 	maxConns            = 2048 // TODO: an arbitrary default number, should be revisited after monitoring
 )
 
+var connTimeout = 5 * time.Second
```

Comment:
Can we make it a constant?

Reply:
It also seem to be used in only one place. Maybe we can define it inside the `ServeHTTP` package directly